frontend/src/style/style.scss

```diff
@@ -0,0 +1,248 @@
+  }
+}
+
+.responsiveImg {
+  max-width: 100%;
+}
+
+.heroHeadline {
+  font-size: $font-size--large-headline--mobile;
+  text-align: center;
+  color: $highlight1;
+  line-height: 4rem;
+
+  @media all and (min-width: 768px) and (min-height: 570px) {
+    margin-top: 0;
+    font-size: $font-size--large-headline--desktop;
+    line-height: 4.4rem;
+  }
+}
+
+.heroText {
+  font-size: $font-size--large-text--mobile;
+  text-align: justify;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  @media all and (min-width: 768px) and (min-height: 570px) {
+    font-size: $font-size--large-text--desktop;
+  }
+}
+
+.heroList {
+  font-size: $font-size--large-text--mobile;
+  line-height: 2rem;
+
+  a {
+    color: $highlight2;
+    text-decoration: none;
+
+    &:hover,
+    &:active,
+    &:visited {
+      color: $highlight2;
+    }
+  }
+}
+
+.logo {
+  margin: 2rem 0;
+}
+
+.logofill {
+  fill: $dark;
+}
+
+.firstSlide {
+  position: relative;
+  color: $light;
+  background-color: $dark;
+
+  @media all and (min-height: 570px) {
+    min-height: 80vh;
+  }
+
+  &CenterVert {
+    @media all and (min-height: 570px) {
+      position: absolute;
+      top: 50%;
+      transform: translateY(-50%);
+      width: 100%;
+    }
+  }
+
+  &CenterHoriz {
+    max-width: 460px;
+    margin: auto;
+    padding: 20px;
+  }
+}
+
+.secondSlide {
+  max-width: 768px;
+  margin: auto;
+  padding: 20px;
+  font-size: 1.2rem;
+}
+
+.examples {
+  padding: 1rem;
+  background-color: $white;
+
+  &Demo {
+    @extend .examples;
+    background-color: $highlight2;
+    h2, h3 {
+      color: $white;
+    }
+  }
+
+  &Inner {
+    max-width: 100rem;
+    margin: auto;
+
+    &Center {
+      @extend .examplesInner;
+      text-align: center;
+    }
+  }
+}
+
+.example {
+  .headline {
+    text-align: left;
+  }
+}
+
+.headline {
+  text-align: center;
+  color: $highlight1;
+}
+
+.select {
+  width: 300px;
+}
+
+.slider {
+  max-width: 800px;
+}
+
+.externalLink {
+  vertical-align: baseline;
+  fill: $highlight2;
+  margin: 0 0.4rem;
+  width: 1.4rem;
+  height: 1.4rem;
+  position: relative;
+  top: 0.2rem;
+}
+
+.verticalSlider {
+  width: 100%;
+  max-width: 300px;
+}
+
+.express {
+  text-align: center;
+  padding: 2rem;
+}
+
+
+.flex-container {
+  position: relative;
+  display: flex;
+}
+
+.img1-wrap {
+  position: relative;
+  overflow: hidden;
+  width: 100%;
+  height: 100%;
+}
+
+.image {
+  width: 100%;
+}
+
+.overlay {
+  display: block;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  overflow: hidden;
+  /*width: 100%;*/
+  height: 20%;
+  transition: .5s ease;
+}
+
+.img1-wrap:hover .overlay {
+  height: 100%;
+}
+
+.text {
+  color: black;
+  font-size: 20px;
+  position: absolute;
+  overflow: hidden;
+  top: 50%;
+  left: 50%;
+  height: 45%;
+  width: 100%;
+  text-align: center;
+  padding: 0px 5px 0px 5px;
+  overflow-wrap: break-word;
+  transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
+}
```

Comment:
I understand this css file is taken from another code.
Using it as our main css file would make further development a bit more complicated.
Could we easily discard the things we don't need here?

Reply:
done

Reply:
done